sources/packages/web/src/components/common/AssessmentAwardDetails.vue

```diff
@@ -179,11 +179,11 @@
 import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
 import { AwardDetailsAPIOutDTO } from "@/services/http/dto";
 import { COEStatus, StatusInfo } from "@/types";
-import { PropType, computed, defineComponent } from "vue";
+import { PropType, computed } from "vue";
 import AwardTable from "@/components/common/AwardTable.vue";
 import StatusInfoEnrolment from "@/components/common/StatusInfoEnrolment.vue";
 
-export default defineComponent({
+export default {
```

Comment:
Can you please elaborate on why are we removing the `defineComponent` here?

Reply:
it was because, in m local, I saw a lot of property type errors like this (in a lot of files), 
![image](https://user-images.githubusercontent.com/77353155/195954427-6e7cd82e-c5d7-4413-8adf-4ecff3d358da.png). and I know we do have a plan to have a ticket for `defineComponent` so, at that time maybe the dev can work on it too (sp that this file won't get missed). maybe i can add it back
![image](https://user-images.githubusercontent.com/77353155/195954611-dd323b42-80cc-4821-8624-156c945c8879.png)

Reply:
mm... I haven't created it. if the devs haven't created any, i will create one

Reply:
created [defineComponent](https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1461)